web-client/src/components/AddSockModal.tsx

```diff
@@ -0,0 +1,30 @@
+import { Dialog, DialogContent, DialogHeader, DialogTitle } from "@/components/ui/dialog";
+import AddSockForm from "./AddSockForm";
+import { Sock } from "@/api/inventory/model";
+
+interface AddSockModalProps {
+    isOpen: boolean;
+    onClose: () => void;
+    onAddSock: (sock: Sock) => void;
+}
+
+const AddSockModal: React.FC<AddSockModalProps> = ({ isOpen, onClose, onAddSock }) => {
+    return (
+        <Dialog open={isOpen} onOpenChange={onClose}>
+            <DialogContent className="max-w-2xl" aria-describedby="add-sock-description">
+                <DialogHeader>
```

Comment:
Update the dialog header to contain this:

```ts
<DialogHeader>
  <DialogTitle>Add new sock</DialogTitle>
  <DialogDescription
    id="add-sock-description"
    className="text-sm text-muted-foreground"
  >
    Fill out the form below to add a new sock to the inventory.
  </DialogDescription>
</DialogHeader>
```

Basically moves the description up and modifies some styles